README.md

```diff
@@ -68,6 +68,18 @@ Include the Javascript in your base template
 ```html
   <script type="text/javascript" src="{% static 'wagtailnhsukfrontend/js/nhsuk-4.1.0.min.js' %}" defer></script>
 ```
+## Upgrading
```

Comment:
I don't think this "upgrading" is necessary in the readme. Having notes in the changelog should be enough.
Maybe a small section that says "If you are using v0 and wish to upgrade to v1, see the changelog" with a link to the changelog "upgrade considerations" notes.